client/components/d3chart/index.js

```diff
@@ -130,9 +124,11 @@ class D3Chart extends Component {
 		const xScale = getXScale( uniqueDates, adjWidth );
 		const xTicks = getXTicks( uniqueDates, adjWidth, mode, interval );
 		return {
+			adjHeight,
+			adjWidth,
 			colorScheme,
 			dateSpaces: getDateSpaces( data, uniqueDates, adjWidth, xLineScale ),
-			height: calculatedHeight,
+			height,
```

Comment:
this is unused.

Reply:
Removed! :+1: 

16ec7bdf67893b849d554fd5afe4060f8bad205f